easybuild/easyconfigs/p/PaStiX/pastix-intel.patch

```diff
@@ -0,0 +1,12 @@
+diff -Nru pastix-6.0.0-orig/cmake_modules/morse_cmake/modules/find/FindBLAS.cmake pastix-6.0.0/cmake_modules/morse_cmake/modules/find/FindBLAS.cmake
```

Comment:
please include patch description + mention author (any lines above this line are comments)

and then don't forget to update the checksum for the patch file